tests/test_monitors.py

```diff
@@ -138,6 +148,55 @@ def test_log_monitor_should_pass(make_data):
     assert runner.result.monitor_results[0].error is None
 
 
+def test_warning_monitor_should_fail(make_data):
+    """ WarningCount should fail if the # of warning log messages exceed the limit """
+
+    # Scenario # 1
+    data = make_data({SPIDERMON_MAX_WARNINGS: 0})
+    runner = data.pop("runner")
+    suite = new_suite([WarningCountMonitor])
+    data["stats"]["log_count/WARNING"] = 2
+    runner.run(suite, **data)
+    assert "Found 2 warnings in log" in runner.result.monitor_results[0].error
+
+    # Scenario # 2
+    data = make_data({SPIDERMON_MAX_WARNINGS: 10})
+    runner = data.pop("runner")
+    suite = new_suite([WarningCountMonitor])
+    data["stats"]["log_count/WARNING"] = 12
+    runner.run(suite, **data)
+    assert "Found 12 warnings in log" in runner.result.monitor_results[0].error
+
+
+def test_warning_monitor_should_pass(make_data):
```

Comment:
**issue**: This tests if the monitor doesn't fail if disabled. Comment is wrong.